tests/test_e2e_20_flow_manager.py

```diff
@@ -397,11 +196,8 @@ def test_020_delete_flow(self):
         }
 
         api_url = KYTOS_API + '/flow_manager/v2/flows/00:00:00:00:00:00:00:01'
-        response = requests.post(api_url, data=json.dumps(payload),
-                                 headers={'Content-type': 'application/json'})
-        assert response.status_code == 200
```

Comment:
I think we should preserve the asserts related to the response and response.json()